templates/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
+            <a href="#">My Logo</a>
+          </div>
+
+          <div class="nav__container__parnav__bars">
+            <div class="nav__container__parnav__bars__responsive">
+              <img
+                src="..//static/imgs/Union@2x.png"
+                alt="Union@2x"
+                class="responsive nav__container__parnav__bars__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <!-- end nav -->
+    <!-- start sectionone -->
+    <section class="sectionone" id="sectionone">
+      <div class="overlay">
+        <div class="sectionone__heroimage">
+          <h2>
+            I’m Alexandra <br />
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through <br />
+            some of my work and experience! If you like what you see and have a
+            project you need <br />
+            coded, don’t hestiate to contact me.
+          </p>
+          <span>LET’S CONNECT</span>
```

Comment:
Kindly note that this span should be a `button` or `a` tag 🤗
- _Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href._